met-api/src/met_api/services/widget_service.py

```diff
@@ -92,3 +92,11 @@ def save_widget_items_bulk(self, widget_items: list, widget_id, user_id):
         self.create_added_widget_items(widget_items, widget_items_db, user_id)
         self.update_widget_items_sorting(widget_items, widget_id, user_id)
         return widget_items
+
+    @staticmethod
+    def delete_widget(engagement_id, widget_id):
+        """Remove widget from engagement."""
+        updated_widgets = Widget.remove_widget(engagement_id, widget_id)
```

Comment:
updated_widgets?

Reply:
i was doing updating and removing simultaneously so just cleaned that up